spec/classes/foreman_proxy__plugin__dynflow_spec.rb

```diff
@@ -42,6 +42,23 @@
     end
   end
 
+  describe 'with open file limits defined' do
+    let :facts do
```

Comment:
I'd leave this out and modify the expectations to either ```should``` (non-Fedora RH) or ```should_not``` (everything else).

Reply:
@ekohl i do not know what you mean with this, I dont see an example.

Reply:
Similar to: https://github.com/theforeman/puppet-foreman_proxy/blob/feb4b4e12bd5febca60e3430c8b920c40707efb5/spec/classes/foreman_proxy__tftp_spec.rb#L93-L95

Then add an else statement with ```should_not```. That way you test all operating systems behave correctly when open file limits is defined rather than only RH 7.